src/tools/fineCalibrationChecker/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+# Copyright: (C) 2025 iCub Facility - Istituto Italiano di Tecnologia
+# Authors: Jacopo Losi
+# CopyPolicy: Released under the terms of the GNU GPL v2.0.
+
+project(fineCalibrationChecker)
+# find YARP
+find_package(YARP COMPONENTS os sig dev)
+find_package(iCubDev 2.7.0 QUIET)
```

Comment:
Sorry I missed that during the review, but probably this CMakelists has to be revised a bit in particular how we handle the dependency from openCV.

We will see together once I come back from vacation

/remind 26th June